shopify/slack-notification-template/web/routes/index.tsx

```diff
@@ -142,7 +151,12 @@ const SlackChannelSelectionForm = ({
                       label="Select a channel"
                       value={inputValue}
                       placeholder={
-                        channels?.filter(
+                        (
+                          channels as {
```

Comment:
if we repeat this type, we might as well define it separately

Reply:
Added a type